### ISSUE TITLE
Windows compatibility

### ISSUE THREAD

**Issue by minhdng** [statius/RunningCouplings#1]
Problem at loading the package 

![loading](https://user-images.githubusercontent.com/21085760/93407300-6f37ea00-f857-11ea-81ed-2485cafc7753.PNG)

Operating system: Windows 10 Pro 64 bit (10.0, Build 18363)

It seems that some code are written in Unix, thus the Windows compatibility.

**Comment by statius** [comment]
Yes, it looks like this is a problem with SMDR, which uses the GNU `make` utility to compile. I believe this is available on Windows (cf. https://stackoverflow.com/questions/32127524), but it will need to be installed first.

What concerns me here is that the _RunningCouplings_ package reported a successful SMDR build. This is due to the fact that the code does not throw an exception for a missing `make` command (i.e., for an error from `RunProcess` rather than just a nonzero exit code from `make`).

**Comment by statius** [comment]
See #3.

**Comment by minhdng** [comment]
`make` installed through MinGW with `PATH` set up, but still reporting error while compiling SMDR.

![error](https://user-images.githubusercontent.com/21085760/93549407-d379ad00-f92e-11ea-81ab-f6e8bc4d759b.PNG)

It seems like there are other compatible places in the code.

**Comment by statius** [comment]
Yes, it looks some of the other Unix utilities will also be necessary, unless you want to edit the SMDR makefile directly.
